docs/src/examples/Optimal Control/quadcopter.jl

```diff
@@ -0,0 +1,140 @@
+# # Quadcopter Trajectory and Set Point Tracking
+
+# The quadcopter is an unmanned aerial vehicle with 4 propellors and in this case study,
+# we seek to determine an optimal control policy for the 
+# 4 control inputs that must be adjusted over a specified time period to enable the 
+# quadcopter to closely follow given set points/trajectory while minimizing the difference between 
+# the set point and actual position and propellor input
```

Comment:
I would clean up the grammar here a bit and be more succinct. This is a flight path tracking problem for a quadcopter UAV.